Resolve conflicts from main

TalhaMaliktz opened the pull request.

IkerAlus commented on introduction.md:
```diff
@@ -20,3 +19,16 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+It consists of rules about the [P2P communication](p2p-communication.md), block and transaction formats, the [consensus algorithm](consensus.md), and of rules how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
+
+The Lisk protocol is constantly evolving. The discussions about planned changed take place at [research.lisk.io](https://research.lisk.io).
```
`The discussions about planned changed...`
->
The discussions about the planned changes.

IkerAlus commented on introduction.md:
```diff
@@ -20,3 +19,16 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+The protocol defines these rules without a focus on concrete implementation, and it is, therefore, possible to implement it in any compatible programming language.
+
+Currently, the official implementations of the Lisk protocol are [Lisk Core](../lisk-core/introduction.md), [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) and [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md), which are written in Javascript and Typescript.
```
I assume that the path for these three links is going to change (now we don't have "lisk-sdk" folder in the docs repo. So please double check that they are correct because I can't from this file in this branch

Tschakki replied:
We first can check this once everything is merged into `development`

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+## The Lisk Protocol
+
+The Lisk Protocol is a formal set of rules of the Lisk blockchain.
```
```suggestion
The Lisk Protocol is the formal set of rules of the Lisk blockchain.
```

janhack replied:
```suggestion
The Lisk Protocol is the formal set of rules of the Lisk blockchain.
```

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+The Lisk Protocol is a formal set of rules of the Lisk blockchain.
+
+It consists of rules about the [P2P communication](p2p-communication.md), block and transaction formats, the [consensus algorithm](consensus.md), and of rules how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
```
```suggestion
It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
```

janhack replied:
Slightly modified the formulation and added links to the `blocks` and `transactions` section.

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+It consists of rules about the [P2P communication](p2p-communication.md), block and transaction formats, the [consensus algorithm](consensus.md), and of rules how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
+
+The Lisk protocol is constantly evolving. The discussions about planned changes take place at [research.lisk.io](https://research.lisk.io).
```
```suggestion
The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Discussions about concrete proposed changes to the Lisk protocol take place openly and transparently at [Lisk Research](https://research.lisk.io).
```

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+The Lisk protocol is constantly evolving. The discussions about planned changes take place at [research.lisk.io](https://research.lisk.io).
+The result of the discussions is reflected in LIPs (Lisk Improvement Proposals) in the [LIPs Github repository](https://github.com/LiskHQ/lips) and later as part of documentation on official [Lisk documentation](https://docs.lisk.io).
+
+The protocol defines these rules without a focus on concrete implementation, and it is, therefore, possible to implement it in any compatible programming language.
```
I would maybe remove `compatible`, because I don't understand how a programming language can be compatible or not.

Tschakki replied:
hm yeah, but it might not be 100% correct to say "in every programming language" as well.
There might be some turing incomplete languages, that cannot implement everything the protocol describes.

But beside this I imagine it's theoretically possible with every turing complete language.

janhack replied:
That makes sense. What about `common` instead?

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+It consists of rules about the [P2P communication](p2p-communication.md), block and transaction formats, the [consensus algorithm](consensus.md), and of rules how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
+
+The Lisk protocol is constantly evolving. The discussions about planned changes take place at [research.lisk.io](https://research.lisk.io).
+The result of the discussions is reflected in LIPs (Lisk Improvement Proposals) in the [LIPs Github repository](https://github.com/LiskHQ/lips) and later as part of documentation on official [Lisk documentation](https://docs.lisk.io).
```
After a LIP comes into effect, i.e., the protocol changes have been adopted by the Lisk blockchain, the official [Lisk documentation](https://docs.lisk.io) will be updated accordingly.

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
+
+The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Discussions about changes to the Lisk protocol take place openly and transparently at [Lisk Research](https://research.lisk.io).
```
```suggestion
The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Changes of the Lisk protocol are formally specified as Lisk Improvement Proposals (LIPs) and published in the [LIPs Github repository](https://github.com/LiskHQ/lips). The process regarding LIPs is described in detail in the [LIP purpose and guidelines](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0001.md) and discussions about possible LIPs take place openly and transparently at [Lisk Research](https://research.lisk.io).
```

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
+
+The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Discussions about changes to the Lisk protocol take place openly and transparently at [Lisk Research](https://research.lisk.io).
+The result of the discussions is reflected in LIPs (Lisk Improvement Proposals) in the [LIPs Github repository](https://github.com/LiskHQ/lips) and later as part of documentation on official [Lisk documentation](https://docs.lisk.io).
```
```suggestion
After a LIP comes into effect, i.e., the protocol changes have been adopted by the Lisk blockchain, the official Lisk documentation will be updated accordingly.
```

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+The Lisk Protocol is a formal set of rules of the Lisk blockchain.
+
+It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
```
```suggestion
It defines rules for the [peer-to-peer communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
```

janhack commented on introduction.md:
```diff
@@ -20,3 +19,19 @@ The utility token used in the Lisk ecosystem is known as Lisk (LSK).
+
+It defines the rules for the [P2P communication](p2p-communication.md), [block](blocks.md) and [transaction](transactions.md) formats, the [consensus algorithm](consensus.md), and how [signatures](https://lisk.io/academy/blockchain-basics/how-does-blockchain-work/digital-signatures) and [IDs](security.md) are computed and verified.
+
+The Lisk protocol is in the process of evolution along the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Changes of the Lisk protocol are formally specified as Lisk Improvement Proposals (LIPs) and published in the [LIPs Github repository](https://github.com/LiskHQ/lips). The process regarding LIPs is described in detail in the [LIP purpose and guidelines](https://github.com ...
```
```suggestion
The Lisk protocol is evolving following the objectives outlined by the [Lisk roadmap](https://lisk.io/roadmap). Changes of the Lisk protocol are formally specified as Lisk Improvement Proposals (LIPs) and published in the [LIPs Github repository](https://github.com/LiskHQ/lips). The process regarding LIPs is described in detail in the [LIP purpose and guidelines](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0001.md) and discussions about possible LIPs take place openly and transparently at [Lisk Research](https://research.lisk.io).
```

Tschakki committed:
Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

Tschakki committed:
Docs Release 28-06-2024 (#160)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add info about voting power calculation (#142)

* Add info about voting power calculation

* Add link

* Update link

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Apply feedback

* Add formulas for L1 fee (#138)

* Add formulas for l1 fee

* Apply feedback

* Add fee params

* Add gas tracker section

* Apply feedback

* Apply suggestions from code review

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply feedback

* Update docs/about-lisk/fees.md

* Apply feedback

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Add coming soons (#159)

* Add coming soons

* Make bold

* Fix dev branch (#161)

* Release Docs 14-June-2024 (#151)

* Add new section in docs

* Replace logos - update styling, configuration

* Create sidebar content, update config and test draft

* Groom docs and re-arrange content

Move Governance section to user docs
Style links
Add banners to light or dark mode

* Add hover styling

* Update intro.mdx

* Update intro.md

* Test no jekyll environment

* Update config

* Add Relay Bridge and Superbridge (#139)

* Add bridges

* Add and vice versa

* Update docs/lisk-tools/bridges.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Update contracts.md

* Revamp style and structure of docs

* Update style

* Update links, style, text

* Update text and links at homepage

* Update index.js

* Update src/pages/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update src/pages/index.js

* Add btns, style text, images, btns

* Refine landing page, styles and images

* Update index.js

* Update storage requirements (#148)

* Update intro pages and styles

* Update index.js

* Update dark-more images

* Add highlights to the user-docs intro page.

* Remove docs plugin, add multiple sidebars, update links, set landing page

* Add Tally links (#145)

* Add links

* Update docs/governance/overview.mdx

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Delete userSidebar.js

* Update docusaurus.config.js

* Update docusaurus.config.js

* Update src/components/HomepageFeatures/index.js

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>

* Update legacy chain service info (#149)

* Update legacy chain service info

* Add info about account initialization

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove image

* Create new section for Lisk L1 legacy docs

---------

Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>

---------

Co-authored-by: Mona Bärenfänger <tschakki@posteo.de>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

* Remove line

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>

---------

Co-authored-by: Muhammad Talha <13951043+TalhaMaliktz@users.noreply.github.com>
Co-authored-by: Sameer <sameer.subudhi@lightcurve.io>
Co-authored-by: shuse2 <shus.toda@gmail.com>

Tschakki committed:
Release 05-07-2024 (#167)

Guide on Importing LSK to MetaMask
Add faucet URLs
Add exchanges page

Tschakki committed:
Release 15-07-2024 (#170)

### What was the problem?

This release includes

- #42 
- #154

Tschakki committed:
Release 23-07-2024 (#178)

### Inlcudes
 
- #174 
- #175 
- #176 
- #166

Tschakki committed:
Release 01-08-2024 (#183)

Includes:

- #157

Tschakki committed:
Release 05-08-2024 (#186)

Includes

- #173 
- #177

Tschakki committed:
Release 08.08.2024 (#189)

Includes:

- #180

Tschakki committed:
Release 28-08-2024 (#194)

### Includes 

- #187 
- #191

Tschakki committed:
Release 11-09-2024 (#207)

### Includes

- #99
- #181 
- #196
- #197  
- #204

Tschakki committed:
Release 19-09-2024 (#213)

### Includes

- #203 
- #205 
- #209

Tschakki committed:
Release 26-09-2024 (#219)

### Includes

- #202 
- #199

Tschakki committed:
Release 07-10-2024 (#221)

### Includes

- Add wallets page #217

Tschakki committed:
Release 15-10-2024 (#230)

### Includes

- #225

Tschakki committed:
Release 16-10-2024 (#231)

### Includes

- #182

TalhaMaliktz committed:
Release 12-11-2024 (#236)

[LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

TalhaMaliktz committed:
Release 2: 12-11-2024 (#238)

* [LISKDOCS-14] Update Docs UI to match the Lisk Rebranding (#235)

* Replace new fonts, update colors, etc.

* Apply changes to styling in light and dark mode. Update images

* Replace homepage images and style them

* Update logos, footer, images and buttons' styling

* Apply Rita Osokina's feedback

* Apply Dominic's feedback

* Apply Mona's feedback.

* Apply Mona's feedback.

* Update padding of 'heroBanner'

* [LISKDOCS-23] Update favicon (#237)

Update favicon

TalhaMaliktz committed:
Remove comment and coming soon

TalhaMaliktz committed:
Merge branch 'main' into Resolve-conflicts-from-main
